syntax = "proto3";

option java_package = "com.jiagu.uav.client";
option java_outer_classname = "AckMessage";
option go_package = "./message";

// CMD_ACK  = 0xFEFE
// CMD_SACK = 0xEFEF

message AckProto {
    repeated uint64 seq = 1;
}

message SAckProto {
    string droneId = 1;
    uint64 seq = 2;

    // return code, 0: success, 1: received, < 0: error
    int32 code = 3;
}
